dist/tools/openocd/openocd.sh

```diff
@@ -191,8 +191,16 @@ do_debug() {
     test_elffile
     test_ports
     test_tui
+    # setsid is needed so that Ctrl+C in GDB doesn't kill OpenOCD
+    [ -n "$(which setsid)" ] && SETSID=setsid
```

Comment:
I see. And I realized that the checking isn't needed in my solution, because `SETSID="$(which setsid)"` already does it.